### PR TITLE
Mfa

### PR DESCRIPTION
**Pull request by anthonybrookfield**
initial draft of tips for avoiding reauthentications with MFA

### CODE REVIEW

File: hpc/mfa.rst [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+Many SSH clients can reuse existing SSH sessions for new connections without the need to reconnect.  Some 
+clients also allow sessions to persist temporarily after you have closed all your terminal windows to allow
+you to easily reconnect for a short time without having to reauthenticate.
+
+SSH clients
+^^^^^^^^^^^
+
+SSH has a built in functionality to reuse existing connections for new sessions.  You can enable this feature by adding the following config
+to your `~/.ssh/config` file ::
```
Comment (willfurnass) [inline hpc/mfa.rst:19]: "(On your local machine, not on HPC)"

File: hpc/mfa.rst [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+  Host iceberg.shef.ac.uk
+    ControlMaster auto
+    ControlPath ~/.ssh/sockets/%r@%h-%p
+    ControlPersist 600
+
+You will need to create the directory `~/.ssh/sockets` before running ssh.  The ``ControlPersist`` option allows you to specify how long (in seconds) your SSH connection
```
Comment (willfurnass) [inline hpc/mfa.rst:26]: Need extra backticks around `~/.ssh/sockets`

File: hpc/mfa.rst [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+You can configure Putty to ``Share SSH connections if possible`` via the ``SSH`` option in the ``Connection Catagory`` when configuring a new connection.
+
+As long as your existing connection remains active you can start new sessions without reautenticating by using ``Duplicate Session`` command to start new sessions.
+
+Other applications which use Putty for SSH connections can also re-use your existing connection without needing to reauthenticate.
+
+
+.. note::
+
+    If you perform a large file stranfer over a shared session you may find that other sessions sharing the same connection become less responsive.
```
Comment (willfurnass) [inline hpc/mfa.rst:52]: typo

File: hpc/mfa.rst [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+
+TMUX/screen
+-----------
+
+`TMUX <https://github.com/tmux/tmux/wiki>`_ and `screen <https://www.gnu.org/software/screen/manual/screen.html>`_ are available on the HPC login nodes and 
+can be used to run multiple shell sessions within a single SSH session.
```
Comment (willfurnass) [inline hpc/mfa.rst:59]: Could link to https://rse.shef.ac.uk/blog/tmux-intro/ for more info

File: hpc/mfa.rst (hunk 3 above)
Comment (anthonybrookfield) [reply hpc/mfa.rst]: fixed

File: hpc/mfa.rst (hunk 2 above)
Comment (anthonybrookfield) [reply hpc/mfa.rst]: back ticked

File: hpc/mfa.rst (hunk 4 above)
Comment (anthonybrookfield) [reply hpc/mfa.rst:59]: added link

**Pull request merged**